index.rst

```diff
@@ -21,6 +21,12 @@
                <h2>News</h2>
 
                <ul>
+               <li>
+                  2020-06-23:
+                  <b>GMT postdoctoral position</b> available at SOEST, U of Hawaii at Mānoa.
+                  Click to <a href="LINK TO GOOGLE DOCS HERE"> view ad details</a> or
```

Comment:
Need to add the link to google docs here.

Reply:
https://docs.google.com/document/d/15-ksiUotquaNYVnnpy8NKIWNU8pGPduf8__jZLSvGQE/edit?usp=sharing

Reply:
Done, but not sure it the permission is correct, since I can still edit it.

Reply:
Sorry, this is apparently the direct link to the job ad
https://hcmweb.rcuh.com/psp/hcmprd_exapp/EMPLOYEE/HRMS/c/HRS_HRAM.HRS_APP_SCHJOB.GBL?Page=HRS_APP_JBPST&Action=U&FOCUS=Applicant&SiteId=3&JobOpeningId=220288&PostingSeq=1
The google docs has this link now.